resources/kcp/charts/kyma-metrics-collector/templates/service-monitor.yaml

```diff
@@ -18,7 +18,7 @@ spec:
 {{ include "kyma-metrics-collector.labels" . | indent 8 }}
 {{- end }}
 
-{{- if and (.Capabilities.APIVersions.Has "operator.victoriametrics.com/v1beta1/VMServiceScrape") .Values.vmscrapes.enabled .Values.global.kyma_metrics_collector.enabled }}
```

Comment:
Why is this fragment removed?

Reply:
Because it was a totally unnecessary addition of a recent change (check git history), the whole subchart is conditionally enabled by this value:
https://github.com/kyma-project/control-plane/blob/main/resources/kcp/Chart.yaml#L18

So it doesn't make any sends to conditionally render parts of the subchart.